<launch> 
  <node pkg="scout" type="scout" name="scout">
    <!--param name="port" value="/dev/cu.usbserial-A7004mS2" /-->
    <!--param name="port" value="/dev/ttyUSB0" /-->
	 <param name="sim" value="true" />
  </node>


  <!--
  <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
    <param name="freq" value="10.0"/>
    <param name="sensor_timeout" value="1.0"/>
    <param name="publish_tf" value="true"/>
    <param name="odom_used" value="true"/>
    <param name="imu_used" value="true"/>
    <param name="vo_used" value="false"/>
  </node>
  -->
 </launch>
